PROVESFlightControllerReference/test/int/common.py

```
@@ -53,6 +53,35 @@ def set_radio_recover_fn(fn: Callable[[], None] | None) -> None:
+    is bypass-listed so it works even while desynced. Ground being ahead is
+    normal and left untouched.
+    """
+    fprime_test_api.clear_histories()
+    fprime_test_api.send_command(f"{deframer}.GET_SEQ_NUM")
+    evt = fprime_test_api.await_event(f"{deframer}.SequenceNumberGet", timeout=5)
+    if evt is None:
+        return
+    board_seq = int(evt.args[0].val)
+    seq_file = "./Framing/src/sequence_number.bin"
+    try:
+        with open(seq_file, "r", encoding="utf-8") as f:
```

Ruff (0.15.21)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)